aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -685,7 +684,7 @@ module aptos_experimental::market {
                         order_id: maker_order_id,
                         user: maker_address,
                         orig_size: maker_order.get_orig_size(),
-                        remaining_size: maker_order.get_remaining_size(),
+                        remaining_size: maker_order.get_remaining_size() + maker_remaining_settled_size,
```

Comment:
Is this change needed?

Reply:
Yes, this was a bug in the code, fixing it.